sdk/tests/test_lusid_drive.py

```diff
@@ -165,5 +166,37 @@ def test_delete_file(self):
         self.assertEqual(None, response)
 
 
+    def test_create_folder(self):
+
+        create_folder_request = create_nested_folder(self.api_factory, "/sdk-tests/create-folder/123/abc")
+
+        get_folder = self.search_api.search(search_body=models.SearchBody(
+            with_path="/sdk-tests/create-folder/123", name="abc"))
+
+        get_folder_path = get_folder.values[0].path
+        get_folder_name = get_folder.values[0].name
+
+        self.assertEqual(get_folder_path, "/sdk-tests/create-folder/123")
+        self.assertEqual(get_folder_name, "abc")
+
+    def test_delete_folder(self):
+
+        create_folder_request = create_nested_folder(self.api_factory, "/sdk-tests-delete-folder/123/abc")
+
+        get_folder_before_delete = self.search_api.search(search_body=models.SearchBody(
+            with_path="/", name="sdk-tests-delete-folder")).values
+
+        delete_folder_request = delete_folder(self.api_factory, "/sdk-tests-delete-folder")
+
+        get_folder_after_delete = self.search_api.search(search_body=models.SearchBody(
+            with_path="/", name="sdk-tests-delete-folder")).values
+
+        self.assertTrue(len(get_folder_before_delete) > 0)
+        self.assertTrue(len(get_folder_after_delete) == 0)
+
+
```

Comment:
You could have some additional test cases here, e.g. what happens if you pass in a pat that does not start with `/`